### PR TITLE
CA-222760: Yangtze Repacked inspur hotfix for TLS Cream: The uploaded…

### PR DESCRIPTION
**Pull request by fillzero**
Putting the cipher defaults into xapi rather than into the config file

### CODE REVIEW

**Comment by fillzero** [comment]
Hi @robhoes ,
Would you kindly review this PR.
Thanks.

File: ocaml/xapi/xapi_globs.ml [hunk 1]
```diff
@@ -919,8 +919,8 @@ let xenopsd_queues = ref ([
 
 let default_xenopsd = ref "org.xen.xapi.xenops.xenlight"
 
-let ciphersuites_good_outbound = ref None
-let ciphersuites_legacy_outbound = ref ""
+let ciphersuites_good_outbound = ref "!EXPORT:RSA+AES128-SHA256"
```
Comment (lindig) [inline ocaml/xapi/xapi_globs.ml]: The type changes here from `string option ref ` to `string ref`. Is there a specific reason to give up the option?

Comment (fillzero) [inline ocaml/xapi/xapi_globs.ml]: Hi @lindig 
Thanks for your comments.
I just do as @thomassa  writes here: https://issues.citrite.net/browse/CA-222760?focusedCommentId=1400413&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1400413

Comment (lindig) [inline ocaml/xapi/xapi_globs.ml]: Thanks. I think it makes sense to put the change request into the commit message.

Comment (fillzero) [inline ocaml/xapi/xapi_globs.ml]: Hi @lindig 
Thanks for your comments.
I added change request into the commits.

**Comment by jonludlam** [comment]
OK, this looks good enough to me.

**Pull request merged**